test/collection.js

```diff
@@ -506,7 +506,7 @@
     };
     collection.url = '/test';
     collection.fetch();
-    this.syncArgs.options.success();
+    this.syncArgs.options.success([]);
```

Comment:
We can remove the check, but I don't think `collection.set()` should clear out the collection...